src/config/db.js

```diff
@@ -1,8 +1,17 @@
 const mongoose = require('mongoose');
 require('../utils/env');
 
-const uri = process.env.MONGODB_URI;
+let uri = process.env.MONGODB_URI;
 
+/**
+ * Uncomment this line of code to use a local MongoDB server
+ */
+
```

Comment:
Perhaps this isn't need as there is no commented code

Reply:
I resolved the conflict in that file and that comment came with it. Do you want me to just clear it??